tests/integration/docusaurus/expectations/expectation_management.py

```diff
@@ -0,0 +1,56 @@
+# setup
+import great_expectations as gx
+from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.expectations.core import (
+    ExpectColumnValuesToBeInSet,
+    ExpectColumnValuesToNotBeNull,
+)
+
+context = gx.get_context()
+suite_name = "test-suite"
+column_a = "a"
+column_b = "b"
+column_c = "c"
+
+# create a suite
+suite = ExpectationSuite(name=suite_name)
+suite = context.suites.add(suite=suite)
```

Comment:
This reassignment feels misleading, given that `suites.add` returns the same object that was passed in. I would change this to either of these:
```
suite = context.suites.add(suite=ExpectationSuite(name=suite_name))
```
or (probably better):
```
suite = ExpectationSuite(name=suite_name)
context.suites.add(suite=suite)
```

Reply:
Passing the object into the `SuiteFactory` mutates it, which I find clearer to understand when the variable is reassigned. I've updated to your first suggestion in [018c3a3](https://github.com/great-expectations/great_expectations/pull/9213/commits/018c3a343dbbefc7dbc5c2359be124e73e82bc84)

Reply:
I agree that the reassignment makes it more intuitive that something has changed.  However, if this method is mutating an object in place, as well as returning that same object, please make sure to include in the docstring for the method that the in-place mutation takes place so that people will be aware that their previous object and "new" object are the same if they reference the API docs.

In general, I think it's considered a more pythonic practice to avoid this potential confusion by having methods mutate _or_ return, but not both.  Just looking at another common mutation, a list's `append` method:
```
>> my_list = []
>> my_list = my_list.append("hi")
>> print(my_list)
>> None
```